pkg/kubelet/reason_cache.go

```diff
@@ -93,8 +93,8 @@ func (c *ReasonCache) Remove(uid types.UID, name string) {
 // whether an error reason is found in the cache. If no error reason is found, empty string will
 // be returned for error reason and error message.
 func (c *ReasonCache) Get(uid types.UID, name string) (error, string, bool) {
-	c.lock.RLock()
-	defer c.lock.RUnlock()
+	c.lock.Lock()
+	defer c.lock.Unlock()
```

Comment:
good catch. Same fix was applied to pkg/controller/lookup_cache.go as well a few days ago.

https://github.com/kubernetes/kubernetes/pull/26758/commits/9a0ff5a9e8a8fe4fb6f6ac0c8b8eb8597c19ff40